src/argowrapper/engine/argo_engine.py

```diff
@@ -698,7 +711,7 @@ def check_user_info_for_billing_id_and_workflow_limit(self, request_token):
             return None, None
 
     def check_user_monthly_workflow_cap(
-        self, request_token, billing_id, custom_limit: int
+        self, request_token, billing_id, custom_limit: Optional[int]
```

Comment:
```suggestion
        self,
        request_token: str,
        billing_id: Optional[int] = None,
        custom_limit: Optional[int] = None
```
in this case, I think this is the right way to type it, as `request_token` is a mandatory string.

Reply:
Nice! 👍